src/Copy/Text.elm

```diff
@@ -198,6 +198,9 @@ t key =
         BackToEventsLinkText ->
             "All events"
 
+        EventVisitPublisherUrlText ->
+            "Visit Publisher's site"
```

Comment:
Do we have a label for the publisher we can use here? Similar to what I did with the events backlink button label in #305. If not, no worries.